tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1772,19 +1775,20 @@ void import_export_opaque_with_data( data_t *data, int type_arg,
     TEST_EQUAL( psa_get_key_type( &got_attributes ), type );
     if( attr_bits != 0 )
```

Comment:
Please avoid double negations: instead of `if( a != b ) F(); else T();`, write `if( a == b ) T(); else F();`